packages/router/src/params.tsx

```diff
@@ -16,29 +14,27 @@ export const ParamsProvider: React.FC = ({ children }) => {
   const { routes, paramTypes } = useRouterState()
   const location = useLocation()
 
-  let initialParams = parseSearch(location.search)
+  let params = parseSearch(location.search)
```

Comment:
```suggestion
  let searchParams = parseSearch(location.search)
```

Reply:
Main reason why I didn't call this searchParams and renamed it from initialParams to just params was to save on naming extra variables and the params context value and it is no longer initial params now that this updates whenever location is updated